### PR TITLE
LO: Adding Localization Pack for Sénégal

### PR DESCRIPTION
**Pull request by mohamadou**
LO:Localization Pack for Sénégal

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       |"1.6.1.x"
| Description?  |Adding new Localization pack for Sénégal.
| Type?         | improvement
| Category?     | LO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | .

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/7803)

### CODE REVIEW

**Comment by mohamadou** [comment]
LO: adding Localization Pack for Sénégal

**Comment by xBorderie** [comment]
Thank you for your contribution!

Sadly, you now have malformed commits instead of just one commit:
* the initial commit, which does not follow the [PS naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message),
* one commit which does follow the naming convention.

Could you try and [squash those commits into just one](http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request)? [Another explanation of squashing](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/).

Thank you!

File: localization/sn.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<localizationPack name="Sénégal">
```
Comment (AlexEven) [inline localization/sn.xml]: Can you please switch to "Senegal" as all localization packs have an English name?

File: localization/sn.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  <currencies>  
+    <currency name="West African CFA franc" iso_code="XOF" iso_code_num="4217" sign="XOF" blank="1" conversion_rate="1" format="1" decimals="0"/>
```
Comment (AlexEven) [inline localization/sn.xml]: Format 1 means "X0,000.00 (Such as with Dollars)"
Is this what should be applied by default? 

For the decimals, 0 means no decimals in the prices, is this ok too?

**Review by AlexEven — COMMENTED** [review]
Thank you @mohamadou for this PR!

**Comment by AlexEven** [comment]
To cherry-pick in 1.7 when all comments are answered and commit name edited.
Thanks!

File: localization/sn.xml (hunk 2 above)
Comment (mohamadou) [inline localization/sn.xml]: No the format is 2
For the decimals it's okay

**Comment by matks** [comment]
Hello,

I'm sorry but PrestaShop 1.6 support ended on June 30, 2019, consequently this PR will not be merged into the project.
Thank you for your contribution.

Please note that if a critical issue is discovered, there is a dedicated GitHub repository: https://github.com/PrestaShop/PrestaShop-1.6 maintained by a team of volunteers.
From this repository, it will be possible to submit pull requests in order to fix critical issues.

You can read more about it on http://build.prestashop.com/news/1.6.1.x-what-s-next/

**Pull request closed without merging**